src/screens/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Home from './Home';
+import About from './About';
+import Faq from './Faq';
+import ContactUs from './ContactUs';
+
+const MainBody = (props) => {
```

Comment:
Typically react web projects would use a router library such as `react-router`. It's definitely good to minimise the number of dependencies we're using, but I think this is a case where there's just so many edge cases and small things which can be missed that it's really worth adding a router dependency.

Some of the issues with the current router (`MainBody`) that would be solved using a fully fledged router:

- Hovering the links in the header doesn't show where the link is going (e.g. in chrome for real links this shows in the bottom left)
- I can't cmd+click (or middle mouse click) to open these links in a new tab
- We're not using real `<a>` tags
- The URL isn't updating
- I can't use the back button
- I think these issues would impact SEO too

We've used react-router in the past and had good experiences with it so I'd recommend looking at that, or if you have any other routers that solve these issues that would be fine too.

Reply:
Currently we are managing as a single page application. thats why we didn't make links. According to you each link will be navigate to separate page, right?

Reply:
When using something like react-router, it automatically intercepts link clicks (if you use their `Link` component) to update the url in the browser using javascript and add pages to the browser history. So it will still be a single page application, but the issues above would be fixed. Does that help?